### DISCUSSION TITLE
HyperHDR working on my Bananapi

### DISCUSSION THREAD

**Discussion by AmbiMod** [awawa-dev/HyperHDR#29] [General]
Hi my friend.

Just to told you that your binaires working on my banana pi zero. The problem now is to make working the usb hat to put the grabber... and test in real condition.

![hyperhdr](https://user-images.githubusercontent.com/26428901/107084019-c6365f80-67f6-11eb-822a-8ff3db6e9752.png)

I keep you informed as i told you. Thanks again.

**Comment by awawa-dev** [comment]
Great! Hope you will succeed with that pin-header extension.
Looks like it could really make life easier... on my Raspberry Pi Zero to make things work I need:
- active USB hub because Pi Zero is to weak to power all the components
- USB to Ethernet adapter, legacy one but it's working somehow at 10Mb
- grabber connected to the hub
- crossed fingers to make Pi discover all the connected devices during booting ;)
Too much devices and cables so testing new release on that device it really painful for me ;) 

Compare to Pi Zero my Orange Pi H2+ is much more powerful, better equipped (at least as a head-less unit) and is also very cheap but unfortunately I can't recommend it for everyday usage due to some strange issues with the temperature: during compilation CPU I hit 85C (was connected using Ethernet cable), then I put radiator on and that reduced temperature to 75C... then I enabled Wifi and again met 85C during compilation. Very capable and cheap unit but definitely very bad designed.  Maybe not every version and model of Orange is affected by this behavior.
